common/components/Button/Button.tsx

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable react/jsx-props-no-spreading */
-import * as React from "react";
+import React from "react";
+import { Button as HButton } from "@headlessui/react";
```

Comment:
Could name it as HeadlessButton or something similar perhaps, not necessary but will look less like a typo (that is should have been Button for instance, instead of HButton)

Reply:
Since "H" is used for other components in other files, we can keep it for now, so it's the same across files